spec/index.bs

```diff
@@ -7582,6 +7582,8 @@ dictionary GPUComputePassDescriptor : GPUObjectDescriptorBase {
         1. |timestampWrite|.{{GPUComputePassTimestampWrite/querySet}}.{{GPUQuerySet/[[descriptor]]}}.{{GPUQuerySetDescriptor/type}} is {{GPUQueryType/"timestamp"}}.
 
         1. |timestampWrite|.{{GPUComputePassTimestampWrite/queryIndex}} &lt; |timestampWrite|.{{GPUComputePassTimestampWrite/querySet}}.{{GPUQuerySet/[[descriptor]]}}.{{GPUQuerySetDescriptor/count}}.
+
+        1. |timestampWrite|.{{GPUComputePassTimestampWrite/location}} is written at most once for each value in {{GPUComputePassTimestampLocation}} in this compute pass.
```

Comment:
The language is slightly difficult to understand. I would write (as a separate step outside the for-each loop):

```suggestion
    1. No two entries in |this|.{{GPUComputePassDescriptor/timestampWrites}} have the same {{GPUComputePassTimestampWrite/location}}.
```

Reply:
Thanks! Updated with correct indentation.

Reply:
It shouldn't be indented; it doesn't need to be inside the loop because it just says "No two entries".